src/odemis/dataio/test/multi_tiff_test.py

```diff
@@ -0,0 +1,98 @@
+
+import datetime
+import glob
+import json
+import logging 
+import os
+import unittest
+
+import numpy
+
+from odemis import model
+from odemis.dataio import tiff
+from odemis.util.dataio import open_acquisition
+
+logging.basicConfig(level=logging.DEBUG)
+
+def write_test_metadata(path: str):
```

Comment:
This function doesn't seem used. Can you add a comment why it's useful... or delete that function?